### PR TITLE
Ruby and React

### PR DESCRIPTION
**Pull request by looh16**
Set up a "Hello world" project as one app with Webpack

### CODE REVIEW

**Comment by sja-thedude** [comment]
Hi @looh16,

I tried contacting you on slack but you were not available. If you can update your readme with the new template given [here](https://github.com/microverseinc/readme-template#readme) in the next 5 mins then I will proceed else invalidate it.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Invalid Code Review Request ❌

Hi @looh16, This is @sja-thedude your code reviewer.

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

You have submitted a project, that doesn't have an updated professional readme. Kindly add it in order to proceed.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+name: Linters
+
+on: pull_request
+
```
Comment (HAMMAS-SALEEM) [inline .github/workflows/linters.yml:4]: - [ ] You are not using correct Gitflow. In Gitlfow you make a development branch from the main/master branch and then you make all your feature branches from development branch and after making changes you merge that feature branch into your development branch 😊

**Review by HAMMAS-SALEEM — COMMENTED** [review]
# Invalid Submission 🔴

## Hey @looh16 :wave:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following some requirements

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .github/workflows/linters.yml (hunk 1 above)
Comment (looh16) [reply .github/workflows/linters.yml:4]: I followed the procedure that you are explaining here, first I created the master branch, then I created the dev branch from master, and the rest of the branches I created from the dev branch.

Comment (looh16) [reply .github/workflows/linters.yml:4]: I'm using correct Gitflow

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @looh16, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights:
- All linters passing
- All requirements completed
- Professional readme

giphy

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by looh16** [comment]
Thanks!

**Pull request merged**